### ISSUE TITLE
Using cogvideox-5b to make text2video inference video with size 9:16 appears blurry

### ISSUE THREAD

**Issue by wen020** [THUDM/CogVideo#572]
### System Info / 系統信息

diffusers latest version

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

'''
import torch
from diffusers import CogVideoXPipeline, CogVideoXDPMScheduler
from diffusers.utils import export_to_video, load_image

prompt = "a woman danceing"
pipe = CogVideoXPipeline.from_pretrained("THUDM/CogVideoX-5b", torch_dtype=torch.bfloat16).to("cuda")
pipe.scheduler = CogVideoXDPMScheduler.from_config(pipe.scheduler.config, timestep_spacing="trailing")

pipe.transformer.to(memory_format=torch.channels_last)

# local_character_model_path = './pytorch_lora_weights.safetensors'
# pipe.load_lora_weights(local_character_model_path)

pipe.enable_sequential_cpu_offload()

video = pipe(
    prompt=prompt,
    num_videos_per_prompt=1,
    num_inference_steps=50,
    num_frames=49,
    width=768, height=432,
    guidance_scale=7.0,
    generator=torch.Generator(device="cuda").manual_seed(42),
).frames[0]

export_to_video(video, "output4.mp4", fps=8)
'''

result

https://github.com/user-attachments/assets/5adb4b30-def2-463e-b7c6-980e174ccf9b

### Expected behavior / 期待表现

I hope the generated video will be clear

**Comment by zRzRzRzRzRzRzR** [comment]
There are several questions

1. Check if your model is freshly cloned from Hugging Face, and whether the sample height and width are 300.
2. Whether to install diffusers from source code
3. Only CogVideoX1.5-5B-I2V supports custom resolutions; the model in your code does not support this. This resolution is not within our supported range, please check the readme, the short side should be 768. This resolution is not within the scope we support, please check the readme, the short side should be 768.

**Comment by wen020** [comment]
ok, i will try it
